pipeline/merger/src/main/scala/uk/ac/wellcome/platform/merger/rules/ItemsRule.scala

```diff
@@ -44,15 +46,21 @@ object ItemsRule extends FieldMergeRule with MergerLogging {
              sources: Seq[TransformedBaseWork]): FieldData = {
       val sierraItems = target.data.items
       val metsItems = sources.flatMap(_.data.items)
-      info(s"Merging METS items from ${describeWorks(sources)}")
+      val metsUrls = metsItems.flatMap(_.locations).collect {
+        case DigitalLocation(url, _, _, _, _, _) => url
+      }
+      debug(s"Merging METS items from ${describeWorks(sources)}")
```

Comment:
Why debug?

Reply:
It was logging so much per merge I thought I'd quiet it down a bit

Reply:
:+1: for `DEBUG`, tests especially are so noisy (can always temporarily set log level to `DEBUG` in config if need to investigate anything further)